docs/_v1_0/3_undertegners_kontaktinfo.md

```diff
@@ -1,32 +1,48 @@
 ---
-identifier: kontaktinfo
-title: Undertegners kontaktinfo
+identifier: identifikator-kontaktinfo
+title: Undertegners identifikator og kontaktinfo
 layout: default
 ---
 
-Undertegner(e) kan adresseres på to måter, med eller uten fødselsnummer.
+Undertegner kan adresseres/identifiseres på ulike vis, basert på om avsender benytter synkron eller asynkron flyt og hvilken informasjon avsender har om undertegner.
 
 ### Adressering med fødselsnummer
 
-Dette gir en sikrere identifisering, da kun riktig person har mulighet til å åpne og signere dokumentet.
+Dette gir en sikker identifisering, da kun riktig person har mulighet til å åpne og signere dokumentet.
 
 _For [asynkrone signeringsoppdrag](#asynkrone-signeringsoppdrag)_: Undertegner må først identifisere seg ved å logge inn i signeringsportalen med ID-porten, BankID, BankID på mobil eller Buypass. Deretter kan hun åpne og signere dokumentet med f.eks. BankID. BankID vil derfor bli brukt 2 ganger.
 
 _For [synkrone signeringsoppdrag](#synkrone-signeringsoppdrag)_: Avsender er ansvarlig for å sende riktig person til riktig lenke i signeringstjenesten (returneres fra tjenesten ved opprettelse av oppdraget). Undertegner kan deretter åpne og signere dokumentet med f.eks. BankID.
 
-Signaturen vil inneholde navn på undertegner og hvilken elektronisk ID som ble brukt, og du kan velge mellom å vise hele fødselsnummeret eller kun fødselsdatoen deres. (Kun tilgjengelig for private avsendere når signeringsoppdraget opprettes i _avsenderportalen_ per 28/8 2017).
+Signaturen vil inneholde navn på undertegner og hvilken elektronisk ID som ble brukt, og du kan velge mellom å vise hele fødselsnummeret eller kun fødselsdatoen deres. (Er per 28/8 2017 kun tilgjengelig for private avsendervirksomheter når signeringsoppdraget opprettes i _avsenderportalen_).
 
 
-### Adressering uten fødselsnummer
+### Adressering med kontaktinfo for asynkrone oppdrag
 
 _Kun tilgjengelig for [asynkrone signeringsoppdrag](#asynkrone-signeringsoppdrag) og private avsendere_.
```

Comment:
Det er kanskje tilstrekkelig å skrive at det kun er tilgjengelig for private avsendere, siden det står i overskriften at det gjelder for asynkrone oppdrag?

Alternativt bør det spesifiseres under neste overskrift at det kun er tilgjengelig for synkrone oppdrag.

Reply:
Enig i første setning, men hva mente du med

>Alternativt bør det spesifiseres under neste overskrift at det kun er tilgjengelig for synkrone oppdrag.

?

Reply:
Åja. Skjønte! Glem det